src/components/TodoContainer.jsx

```diff
@@ -1,9 +1,104 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import AddTodoForm from './AddTodoForm';
+import TodoList from './TodoList';
+
+const TodoContainer = () => {
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [sortOrder, setSortOrder] = useState('asc'); // State for the sort order
+
+  const airtableURL = `https://api.airtable.com/v0/${import.meta.env.VITE_AIRTABLE_BASE_ID}/${import.meta.env.VITE_TABLE_NAME}`;
+
+  useEffect(() => {
+    fetchData(); // Load data at the start
+  }, [sortOrder]); // Runs when the sort order changes
+
+  // Function to load data from the API
+  const fetchData = async () => {
+    try {
+      const response = await fetch(`${airtableURL}?sort[0][field]=title&sort[0][direction]=${sortOrder}`, {
```

Comment:
Great choice using the Airtable API to do the sorting for you. When possible it's a good idea to have the data come in as you intended to display it. This will minimize the strain on your client and will usually keep your code clean and lightweight i.e. you're not writing a bunch of code to implement this.